benchmarks/lasso/solvers/statsmodels.py

```diff
@@ -0,0 +1,30 @@
+from benchopt.base import BaseSolver
+from benchopt.util import safe_import
+
+
+with safe_import() as solver_import:
+    import statsmodels.api as sm
+
+
+class Solver(BaseSolver):
+    name = 'statsmodels'
+
+    install_cmd = 'pip'
+    requirements = ['statsmodels']
+    requirements_import = ['statsmodels']
```

Comment:
You don't need this line I think. You only need to specify this if it is different from `requirements`, as `scikit-learn, sklearn`.

Reply:
I see, I 've removed it thanks.